commands/scs_dice.py

```diff
@@ -392,10 +416,11 @@ def func(self):
                 response = yield("Add an optional comment to describe the roll. ('enter' for none)")
             step_counter, dice_string_list = self.response_parser(response, step_counter, dice_string_list)
         # Once the while loop concludes, send the joined dice_string to CmdHandler and run SCSDice on it.
-        dice_string = "".join(dice_string_list)
+        joined_dice_list = "".join(dice_string_list)
+        final_dice_string = dice_string + joined_dice_list
         # Calling cmdhandler directly feels sketchy, but the roll code is a contrib, so this seems like the simplest
         # approach to integrating a new command, EZRoll, that is dependent on said roll code.
-        cmdhandler(caller, f"roll {dice_string}")
+        cmdhandler(caller, final_dice_string)
 
     def response_parser(self, response, step_counter, dice_string_list):
         # Parse and corner case depending on the step in assembling the dice roll
```

Comment:
Note to self: rename subfunctions for readability. success() can be increment_and_append() and response_parser() can be... something else